### PR TITLE
Build new HPC page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Build new HPC index page

## QA

- Check https://ubuntu-com-12005.demos.haus/hpc against [copy doc](https://docs.google.com/document/d/1d4E1IA2mEu72q277sihNSkl7QRKsU1ezwA3jVgJ0TCE/edit)

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5856

## Screenshots

image

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@carkod I'll look into getting the right logos, but in the meantime could you make these changes please:

- Can you use these logos please, I'll need to create new ones for Firmus and PGS
  - [Tesla](https://assets.ubuntu.com/v1/d3ee88a5-2018-logo-tesla.svg)
  - [Scania](https://assets.ubuntu.com/v1/52e19575-scania-logo.png)
  - [Mercedes](https://assets.ubuntu.com/v1/a3bb7ca4-mercedes-benz-group-logo.png)
  - [Daimler](https://assets.ubuntu.com/v1/aff445c4-daimler-logo.png)
  - [Astra Zeneca](https://assets.ubuntu.com/v1/ef528144-astrazeneca-logo.png)

- For the HPC section, can you arrange it the same as the strips on the homepage please, which brings the image up and widens the two columns below, like this:
image

- In each section with a product name and title, can you change the sizes to h2 and h4 please, to reflect the hierarchy in the copy doc, like this:
image

- Can you make the Openstack logo 3 cols please, and aligned right

Thanks.

**Comment by spencerbygraves** [comment]
@carkod here are the remaining two logos:

[Firmus](https://assets.ubuntu.com/v1/654fe8a7-firmus-logo.png)
[PGS](https://assets.ubuntu.com/v1/0d09fbe7-pgs-logo.png)

Thanks.

**Comment by carkod** [comment]
> In each section with a product name and title, can you change the sizes to h2 and h4 please, to reflect the hierarchy in the copy doc, like this:

For point 3, the product manager wanted the design as in https://ubuntu.com/automotive, which shows a smaller size heading for h2 and bigger size for h4.

**Comment by carkod** [comment]
Also another comment from product manager:

> Would it be possible to make the "Performance through partnerships" section similar to "Kubernetes resources" on https://ubuntu.com/kubernetes
And I wonder if we could do the same for "Solutions for HPC", I think that might help break the page up a bit, and help categorize things.
And instead of gray/white background have white background for both "HPC workload support on Ubuntu" and "Ideal for independent software vendors" and brake those sections up with a strip similar to the last two secitons on https://ubuntu.com/kubernetes

**Comment by carkod** [comment]
@spencerbygraves also do you know what's the latest openstack logo? the one I have is probably not right

**Comment by spencerbygraves** [comment]
@carkod please can you use [this illustration](https://assets.ubuntu.com/v1/3bf0d30e-partner-with-us-light.svg) in the "Performance through partnerships" section.

Thanks.

**Comment by spencerbygraves** [comment]
@minkyngkm @sowasred2012 I think this just needs the last illustration update [mentioned above](https://github.com/canonical/ubuntu.com/pull/12005#issuecomment-1240833585). Cheers.

File: templates/hpc/index.html [hunk 1]
```diff
@@ -0,0 +1,717 @@
+{% extends "openstack/_base_openstack.html" %}
```
Comment (minkyngkm) [inline templates/hpc/index.html:1]: The meta data of `_base_openstack.html` is for openstack pages so i think we might need a new base file for hpc pages.

File: templates/hpc/index.html [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+          <li class="p-list__item is-ticked">Government</li>
+          <li class="p-list__item is-ticked">Healthcare and pharmaceuticals</li>
+          <li class="p-list__item is-ticked">Life sciences</li>
+          <li class="p-list__item is-ticked">Manufacturing and industry</li>
+          <li class="p-list__item is-ticked">Media and entertainment</li>
+          <li class="p-list__item is-ticked">Research and universities</li>
+          <li class="p-list__item is-ticked">Weather</li>
+        </ul>
+        <p>
+          <a class="p-button--positive" href="/hpc/use-cases">Use cases&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/hpc/index.html:186]: I think its better to remove `>`  as it is a button
```suggestion
          <a class="p-button--positive" href="/hpc/use-cases">Use cases</a>

```

File: templates/hpc/index.html [hunk 3]
```diff
@@ -0,0 +1,717 @@
...
+    <div class="row">
+      <div class="col-8">
+        <h2>Solutions for HPC</h2>
+        <p>Embrace the open source mandate in HPC with Canonical.</p>
+        <p>
+          Learn more about how Canonical, a leading provider of commercial support for open source solutions can help you with your challenges as a trusted partner for  high-performance computing and supercomputing.
+        </p>
+        <br />
+        <p>
+          <a class="p-button--positive" href="/hpc/contact-us">Canonical and HPC learn more&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/hpc/index.html:201]:
```suggestion
          <a class="p-button--positive" href="/hpc/contact-us">Canonical and HPC learn more</a>

```

File: templates/hpc/index.html [hunk 4]
```diff
@@ -0,0 +1,717 @@
...
+          Ceph for HPC, AI/ML, and HPDA
+        </p>
+        <p>
+          CephFS delivers a scale-out parallel file system for workloads requiring POSIX support.
+        </p>
+        <p>
+          Deliver archival tiers, where supported, with object storage.
+        </p>
+        <p>
+          <a class="p-button--positive" href="/hpc/contact-us?product=hpc-ceph">Get in touch to learn more&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/hpc/index.html:422]:
```suggestion
          <a class="p-button--positive" href="/hpc/contact-us?product=hpc-ceph">Get in touch to learn more</a>

```

File: templates/hpc/index.html [hunk 5]
```diff
@@ -0,0 +1,717 @@
...
+      <div class="col-8">
+        <h2>
+          Performance through partnerships
+        </h2>
+        <p>
+          Canonical has a strong network of technical alliances, driving platform compatibility on Ubuntu. Close partnerships make sure Ubuntu runs optimally on the CPU architecture and works well with GPUs, FPGA, accelerators and high speed interconnects.
+        </p>
+        <br />
+        <p>
+          <a class="p-button--positive" href="/hpc/contact-us">Learn more about partnerships&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/hpc/index.html:438]:
```suggestion
          <a class="p-button--positive" href="/hpc/contact-us">Learn more about partnerships</a>

```

File: templates/hpc/index.html [hunk 6]
```diff
@@ -0,0 +1,717 @@
...
+      </div>
+      <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/3bf0d30e-partner-with-us-light.svg",
+          alt="",
+          width="200",
+          height="148",
+          hi_def=True,
+          loading="auto|lazy"
```
Comment (minkyngkm) [inline templates/hpc/index.html:449]:
```suggestion
          loading="lazy"

```

**Comment by carkod** [comment]
All done

File: templates/hpc/index.html [hunk 7]
```diff
@@ -0,0 +1,713 @@
...
+    <div class="row">
+      <div class="col-8">
+        <h3 class="p-heading--5">
+          Ubuntu
+        </h3>
+        <p>
+          Ubuntu ships with the latest developer tool chains and is the OS of choice for multiple HPC workloads. Access a greater talent pool to drive innovation.
+        </p>
+        <p>
+          <a class="p-button--positive" href="/hpc/contact-us?product=hpc-isv">Get in touch as an ISV&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/hpc/index.html:669]:
```suggestion
          <a class="p-button--positive" href="/hpc/contact-us?product=hpc-isv">Get in touch as an ISV</a>

```

File: templates/hpc/index.html [hunk 8]
```diff
@@ -0,0 +1,701 @@
...
+        </h2>
+        <p class="p-heading--2">
+          Intel oneAPI HPC toolkit
+        </p>
+        <p>
+          Hit the ground running with Ubuntu and use Intel’s development tool chain to ease compilation and get optimal performance for HPC workloads.
+        </p>
+        <p>
+          Juju charms help you simplify operations by assisting in the installation, maintenance and integration of applications.
+        </p>
```
Comment (minkyngkm) [inline templates/hpc/index.html:524]: This paragraph needs to be removed 
```suggestion

```

**Review by minkyngkm — APPROVED** [review]
Thanks Carlos! I made one comment for the paragraph that needs to be removed. Other than that, it all looks good to me!

**Pull request merged**